models/models.comments.js

```diff
@@ -36,3 +36,13 @@ exports.checkUserExists = (username) => {
     } else return true;
   });
 };
+
+exports.removeCommentByCommentId = (comment_id) => {
+  return Promise.all([db.query('DELETE FROM comments WHERE comment_id = $1 RETURNING *', [comment_id]), comment_id]).then(([{ rows }, comment_id]) => {
+    return Promise.all([db.query('SELECT * FROM comments WHERE comment_id=$1', [comment_id]), rows]).then(([{ rows }, deleteResult]) => {
```

Comment:
We dont actually need this line. RETURNING * in an SQL query will return all the rows that have been affected/deleted. If the rows have a length of 0, then we can assume that the comment didnt exist in the first place, where as if we get a length of 1, then it did exist and has now been deleted.